yaml/043773c5-120a-4c6b-8485-8f1f5c47fd3e.yml

```diff
@@ -8699,3 +8699,4 @@ KnownVulnerableSamples:
         Code Signing CA,1
 Tags:
 - aswArPot.sys
+- avgArPot.sys
```

Comment:
I found this driver in our testing infrastructure being used with a different name

![Screenshot 2023-06-14 at 08 29 41](https://github.com/magicsword-io/LOLDrivers/assets/2851492/d1c78380-f534-475d-9877-1da9345343ff)

You can also see it on VT that it has been used with multiple names.

https://www.virustotal.com/gui/file/0f016c80c4938fbcd47a47409969b3925f54292eba2ce01a8e45222ce8615eb8/telemetry

![Screenshot 2023-06-14 at 08 23 51](https://github.com/magicsword-io/LOLDrivers/assets/2851492/ff91b0e0-55c0-4aad-83e9-9bc46fe352fb)

I fixed it by using different tags but maybe you want to handle it differently?
E.g. by making the "Filename:" field a list?

Reply:
@Neo23x0 I think tags for now works well, if this happens very often we can change the filename to an array as you suggested.